libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -23,16 +23,33 @@
 
 //! Mimic Qt's qApp for easy access to the application instance
 #define ccApp (static_cast<ccApplicationBase *>( QCoreApplication::instance() ))
-
+//! ccApplicationBase
```

Comment:
For me, here we should add a description of what the class does / what's its purpose. We don't need to recall the name of the class (that will appear in the doxygen doc anyway)

Reply:
ok I thought it would from following Doxygen's guide on classes. It was stating to put //! <class name> then /*!....*/ with the brief.